README.md

```diff
@@ -1,43 +1,87 @@
-# bot2
```

Comment:
why rename to Bot2?

Reply:
okay i see you renamed it to match the others but its kinda fine as bot2 tbh

Reply:
probably lower case or upper case is fine.